CHANGELOG.md

```diff
@@ -5,3 +5,7 @@ All notable changes to `laravel-country-state-city-data` will be documented in t
 ## 2.0.0 - 2021-05-12
 
 - "Integrity constraint violation: 1062 Duplicate entry" Issue fixed
```

Comment:
add ISO code with 2 digit and 3 digit

Reply:
I don't understand this part, please elaborate.

Reply:
for example for India, the ISO code is in 2 char IN and 3 char IND for the US in 2 char US and for 3 char USA
you can check [Wikipedia link](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes)

Reply:
Alright, I will look into that. Thanks.